Linux/flash

```diff
@@ -171,43 +171,17 @@ case "${OSTYPE}" in
 esac
 
 while true; do
-  if [[ -z "${disk}" ]]; then
-    # try to find the correct disk of the inserted SD card
-    disk=$(df | grep /media | cut -f1 -d " " | sed -e 's/[0-9]$//'|sed  -e 's/p.*$//' | sort | uniq)
-
-    if [[ $(echo "${disk}"|awk '{print NF}') -gt 1 ]]; then
```

Comment:
This part of the script I want to keep if a SD card is mounted and can be selected automatically. I only thought of the wait loop.

Wow, Linux support is much harder than I thought with that variety of distros and installed packages.